src/pal/src/exception/signal.cpp

```diff
@@ -697,6 +697,42 @@ PAL_ERROR InjectActivationInternal(CorUnix::CPalThread* pThread)
 
 /*++
 Function :
+    signal_ignore_handler
+
+    Simple signal handler which does nothing
+
+Parameters :
+    POSIX signal handler parameter list ("man sigaction" for details)
+
+(no return value)
+--*/
+static void signal_ignore_handler(int code, siginfo_t *siginfo, void *context)
+{
+}
+
+
+void PAL_IgnoreProfileSignal(void)
+{
+#if !HAVE_MACH_EXCEPTIONS && defined(SIGRTMAX)
```

Comment:
@janvorli Configurable would be good.  It may be nice to allow`COMPlus_PerfSignalIgnoreMask` could allow multiple bits to be set, but it would probably be limited to 64 bits (0-63).  Or a comment describing recommended signals.